### PR TITLE
Merchant index

### PR DESCRIPTION
**Pull request by zacisaacson**

### CODE REVIEW

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Admin::MerchantsController < Admin::BaseController
+  def update
+    merchant = Merchant.find(params[:id])
+    merchant.toggle! :enabled?
```
Comment (johnktravers) [inline app/controllers/admin/merchants_controller.rb:4]: I've never used toggle before. Very nice!

Comment (grwthomps) [inline app/controllers/admin/merchants_controller.rb:4]: Nice find, never used `toggle` before!

File: spec/models/merchant_spec.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
     it { should validate_presence_of :city }
     it { should validate_presence_of :state }
     it { should validate_presence_of :zip }
+    it { should validate_presence_of :enabled? }
```
Comment (johnktravers) [inline spec/models/merchant_spec.rb:10]: We should also validate that enabled? can't be nil

File: app/views/merchants/index.html.erb [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+  <section class="container-fluid row justify-content-center">
+    <% @merchants.each do |merchant|%>
+      <section class='card card-body bg-light m-3 p-4 col-md-3', id= 'merchant-<%=merchant.id%>' >
+        <h2 class='text-center'><%= link_to merchant.name, "/merchants/#{merchant.id}" %></h2>
+        <% if current_admin? %>
+          <% if merchant.enabled? %>
+            <p class="enabled mx-auto"> <%= link_to 'Disable', "/admin/merchants/#{merchant.id}", method: :patch, class: 'btn btn-danger mt-3' %> </p>
+          <% else %>
+            <p class="disabled mx-auto"> <%= link_to 'Enable', "/admin/merchants/#{merchant.id}", method: :patch, class: 'btn btn-success mt-3' %> </p>
+          <% end %>
```
Comment (grwthomps) [inline app/views/merchants/index.html.erb:11]: These buttons look really good

**Review by grwthomps — APPROVED** [review]

**Review by johnktravers — COMMENTED** [review]
Looks great! Nice testing

**Pull request merged**